config/initializers/bugsnag.rb

```diff
@@ -0,0 +1,7 @@
+if Rails.application.credentials.bugsnag_api_key.present?
+  Bugsnag.configure do |config|
+    config.api_key = Rails.application.credentials.bugsnag_api_key
+  end
+else
+  Bugsnag.configuration.logger = ::Logger.new("/dev/null")
```

Comment:
If no key, send to abyss.

Reply:
Do we want to scope this to an environment as well? e.g. `if env.production? then...`

Reply:
This is done by the API key call. This will only load if the API key is present, which it should only be for prod.

Reply:
Can we add a comment for that then? I don’t think that would be immediately obvious to a future dev reading this without some nuanced understanding.

Reply:
In meeting today we discussed and thought this would be okay without comment. It is agnostic of the env and is relying on the credentials being available. Think it should be able to follow without comment. We can add one in the future though if we have any confusions.